src/DynamoCore/Library/XmlDocumentationExtensions.cs

```diff
@@ -101,34 +103,55 @@ private static string CleanUpDocString(this string text)
         }
 
         private static string GetMemberElement(FunctionDescriptor function,
-            string suffix, XDocument xml)
+            string property, XmlReader xml, string paramName = "")
```

Comment:
Can you try to remove this `XmlReader xml` parameter? The `function` has an internal reference to `PathManager`, which means we can make the following call to obtain `XmlReader` in `GetMemberElement` method:

```csharp
XmlReader xml = null;
if (!string.IsNullOrEmpty(function.Assembly))
    xml = DocumentationServices.GetForAssembly(function.Assembly, function.pathManager);
```

We have a bigger plan to make `XmlDocumentationExtensions` a non-static class (so that we don't have to keep `PathManager` inside `FunctionDescriptor` because that is weird), but for now, this should work just fine :)